docs/docs/intro.md

```
@@ -1,1983 +1,133 @@
-    * [.getPlayerRankings(year, page, group, state)](#cfb.getPlayerRankings) ⇒
-    * [.getSchoolRankings(year, page)](#cfb.getSchoolRankings) ⇒
-    * [.getSchoolCommits(year, school)](#cfb.getSchoolCommits) ⇒
-    * [.getRankings(year, week)](#cfb.getRankings) ⇒
-    * [.getSchedule(year, month, day, group, seasontype)](#cfb.getSchedule) ⇒
-    * [.getScoreboard(year, month, day, group, seasontype, limit)](#cfb.getScoreboard) ⇒
-    * [.getConferences(year, group)](#cfb.getConferences) ⇒
-    * [.getStandings(year, group)](#cfb.getStandings) ⇒
-    * [.getTeamList(group)](#cfb.getTeamList) ⇒
-    * [.getTeamInfo(id)](#cfb.getTeamInfo) ⇒
-    * [.getTeamPlayers(id)](#cfb.getTeamPlayers)
+<a href='https://js.sportsdataverse.org/'><img src='https://raw.githubusercontent.com/sportsdataverse/sportsdataverse-js/main/docs/static/img/sdv-js-logo.png' align="right" width="20%" min-width="100px"/></a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)